src/tigerbeetle/cli.zig

```diff
@@ -160,15 +159,15 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
         if (mem.startsWith(u8, arg, "--cluster")) {
             if (command != .format) fatal("--cluster: supported only by 'format' command", .{});
             cluster = parse_flag("--cluster", arg);
+        } else if (mem.startsWith(u8, arg, "--replica-count")) {
+            if (command != .format) fatal("--replica-count: supported only by 'format' command", .{});
+            replica_count = parse_flag("--replica-count", arg);
         } else if (mem.startsWith(u8, arg, "--replica")) {
```

Comment:
`--replica-count` needs to go first, becaues `--replica` is a perfix of that. I wonder if we should build our own argparse or something? I noticed that the syntax we use for benchmark's argument is different. 

If we do that, two models we can follow:

* do whatever `./zig/zig` does
* do https://github.com/blyxxyz/lexopt, which is the best model which hits "simple" and "pedantically correct"
* (I also like fuchsia guidelines https://fuchsia.dev/fuchsia-src/development/api/cli#command_line_arguments, but they are hated by folks who are accustomed to gnu-style options).

Reply:
:+1: We have a several ad-hoc cli arg parsers; it would be nice to simplify that.

> `--replica-count` needs to go first

We could do `mem.startsWith(u8, arg, "--replica-count=")` (include the equals sign) though that would mess with `parse_flag`'s error message.